packages/node_modules/samples/browser-meetings/index.html

```diff
@@ -29,18 +34,23 @@
     <h1>THIS IS A WORK IN PROGRESS SAMPLE</h1>
     <section>
       <h2>Meetings</h2>
-      <p>[[APPLICATION DESCRIPTION]]</p>
+      <p>This is the meetings plugin sample for the Webex JS SDK.</p>
```

Comment:
My previous comment meant to have each sentence on its own line, but not necessarily in each own `<p>` tag. This way is the paragraph is more readable from the source but still looks like one paragraph when the page displays.